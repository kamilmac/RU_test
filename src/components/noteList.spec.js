import React from 'react'
import { shallow } from 'enzyme'
import NoteList from './NoteList'

it('renders NoteList', () => {
    const notes = [
        {
            text: 'test'
        },
        {
            text: 'test1'
        },
    ]
  
    const wrapper = shallow(<NoteList notes={notes} textColor="black" />)
    expect(wrapper).toMatchSnapshot()
    expect(wrapper.children().length).toEqual(2)
    expect(wrapper.contains('test')).toEqual(true)
    expect(wrapper.contains('test1')).toEqual(true)
})