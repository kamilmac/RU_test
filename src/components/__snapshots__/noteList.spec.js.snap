exports[`test renders NoteList 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <styled.div
    height={undefined}>
    <styled.p
        bgColor={undefined}
        textColor="black">
        test
    </styled.p>
    <styled.p
        bgColor={undefined}
        textColor="black">
        test1
    </styled.p>
</styled.div>,
  "nodes": Array [
    <styled.div
      height={undefined}>
      <styled.p
            bgColor={undefined}
            textColor="black">
            test
      </styled.p>
      <styled.p
            bgColor={undefined}
            textColor="black">
            test1
      </styled.p>
</styled.div>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 2,
      "_context": Object {},
      "_currentElement": <NoteList
        notes={
                Array [
                        Object {
                          "text": "test",
                        },
                        Object {
                          "text": "test1",
                        },
                      ]
        }
        textColor="black" />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StatelessComponent {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {
          "notes": Array [
            Object {
              "text": "test",
            },
            Object {
              "text": "test1",
            },
          ],
          "textColor": "black",
        },
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <styled.div
          height={undefined}>
          <styled.p
                    bgColor={undefined}
                    textColor="black">
                    test
          </styled.p>
          <styled.p
                    bgColor={undefined}
                    textColor="black">
                    test1
          </styled.p>
</styled.div>,
        "_debugID": 2,
        "_renderedOutput": <styled.div
          height={undefined}>
          <styled.p
                    bgColor={undefined}
                    textColor="black">
                    test
          </styled.p>
          <styled.p
                    bgColor={undefined}
                    textColor="black">
                    test1
          </styled.p>
</styled.div>,
      },
      "_renderedNodeType": 1,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <NoteList
    notes={
        Array [
            Object {
              "text": "test",
            },
            Object {
              "text": "test1",
            },
          ]
    }
    textColor="black" />,
}
`;
